src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -151,13 +151,20 @@ public int parseArguments(Parameters params) throws CmdLineException {
   /** Mappings should be comma separated, with no spaces, one mapping per line */
   String knownClassAliasesFile = null;
 
+  @Option(
+    name = "--aliasedNames",
```

Comment:
aliased referes more the the workaround than the original problem. I would call it "assignedNamespaces" (because foo.bar was supposed to be a namespace, but got assigned a "non-namespace" value. 
Or at least "needAliasedNames".